### PR TITLE
DOC: Added the README file

### LINKED ISSUE #3: Code should have a README document
The README file will help visitors learn about the Add-On and how to run it.

### PR DESCRIPTION
**Pull request by turanoo**
Fixes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Survey Sync
+
+## Google Sheets add-on
+
+This is a Google Sheets add-on that will facilitate the synchronization of data
+from a survey database to a Google Sheets spreadsheet.
+
+### Prereqs
+
+-
```
Comment (thermokarst) [inline README.md]: Can you replace this list with a placeholder that says "To be determined"?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+-
+-
+
+### Developer Quickstart
```
Comment (thermokarst) [inline README.md]: Can you add a note in this section that says "Coming soon"?

**Review by thermokarst — CHANGES_REQUESTED** [review]
Thanks @turan1393! This looks good, but I have a few minor updates I would like you to make, please see the inline comments. Thanks!

File: README.md (hunk 1 above)
Comment (turanoo) [reply README.md]: Yeah I will fix it right-away.

File: README.md (hunk 2 above)
Comment (turanoo) [reply README.md]: Yes, I will make the change. Thanks for the feedback!

**Review by thermokarst — APPROVED** [review]

**Comment by thermokarst** [comment]
Thanks @turan1393!

**Pull request merged**